src/parser/parser.py

```diff
@@ -0,0 +1,64 @@
+"""Parses a PDF document.
+"""
+import logging
+
+
+import slate
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Article:
+    """Class for storing articles.
+    """
+
+    def __init__(self, number, text):
+        self.number = number
+        self.text = text
```

Comment:
This by no means is your responsibility, but I think we should align our attribute names to match throughout the entire application just to reduce friction in the convention.

I highlight this because in the web app, I am calling the body of every article `content`, and I believe that having different names on the different scripts and parts of the program, may be confusing to any external.

Reply:
Yeah we should